### PR TITLE
fix: fix search issues caused by spaces in vendor/product names

### LINKED ISSUE #153: Search Function Does not Work if a Whitespace is included
## Expected Behavior
The search https://www.opencve.io/vendors/cisco/products?search=ios+xe should provide as result
Vendor Cisco and Product "Ios Xe"

## Actual Behavior
Doesn't return any matches. It is the same for all products with a white space, e.g., the same for https://www.opencve.io/vendors/microsoft/products?search=Windows+10

Same issue for vendors: https://www.opencve.io/vendors?search=Fedora+Project

## Steps to Reproduce the Problem
Any product with a white space

### PR DESCRIPTION
**Pull request by M0dEx**
Fixes #153.

P.S.: I apologize for the messed up PRs.

### CODE REVIEW

File: tests/views/test_vendors.py [hunk 1]
```diff
@@ -17,3 +17,25 @@ def test_list_vendors(client, create_vendor):
...
+
+def test_vendor_search(client, create_vendor):
+    create_vendor("the_vendor_1", "product1")
+    create_vendor("the_vendor_2", "product1")
+    create_vendor("the_vendor_2", "product2")
+
+    response = client.get("/vendors?search=The Vendor 1")
+    assert response.status_code == 200
+    assert b"The Vendor 1" in response.data
+    assert b"The Vendor 2" not in response.data
```
Comment (ncrocfer) [inline tests/views/test_vendors.py:41]: Copy-paste of the previous test, you can remove this one :)

File: tests/views/test_products.py [hunk 2]
```diff
@@ -9,5 +9,26 @@ def test_get_products(client, create_vendor):
...
+
+
+def test_product_search(client, create_vendor):
+    create_vendor("the_vendor_1", "the_product_1")
+    create_vendor("the_vendor_1", "the_product_2")
+
+    response = client.get("/vendors/the_vendor_1/products?search=The Product 1")
+    assert response.status_code == 200
+    assert b"The Product 1" in response.data
+    assert b"The Product 2" not in response.data
```
Comment (ncrocfer) [inline tests/views/test_products.py:34]: Copy-paste of the previous test, you can remove this one :)

**Review by ncrocfer — CHANGES_REQUESTED** [review]

File: tests/views/test_vendors.py (hunk 1 above)
Comment (M0dEx) [reply tests/views/test_vendors.py:41]: Oh yeah, how did that get there?!

I could swear I only put it there once.

**Review by ncrocfer — APPROVED** [review]

**Comment by ncrocfer** [comment]
Thanks for this contribution @M0dEx 👍

**Pull request merged**